doc/source/cookbook/test-modulefiles.rst

```diff
@@ -3,11 +3,11 @@
 Testing Modulefiles
 ===================
 
-The following is an example for a ``ModulesTest`` proc of a Modulefile and its output.
-It checks whether the ``TESTDIR`` is a directory, checks that it can enter it,
-and whether a file ``TESTFILE`` can successfully be created there.
+The following is an example for a ``ModulesTest`` subroutine of a Modulefile and its output.
+It checks whether the :file:`TESTDIR` is a directory, checks that it can enter it,
+and whether a file:file:`TESTFILE` can successfully be created there.
```

Comment:
(defect) keyword `file:` is used twice

Reply:
there is a space missing. Thanks for spotting this :+1: